### DISCUSSION TITLE
How can i Become Github Campus Expert.

### DISCUSSION THREAD

**Discussion by MGJillaniMughal** [community/community#21299] [GitHub Education]
Hi Everyone Please tells me how can I become a GitHub Campus Expert?

**Comment by baysideengineer** [comment]
Hey @MGJillaniMughal, I guess everything you need to know can be found on the [Campus Expert](https://education.github.com/experts) site. There is a lot of information about how to apply, what they need from you and how it is to be a Campus Expert. I wish you good luck on your journey.
